examples/3dtiles.html

```diff
@@ -152,15 +152,12 @@
                 }
             }
             function picking(event) {
-                // Pick an object with batch id
-                var mouse = globe.eventToNormalizedCoords(event);
-                var raycaster = new itowns.THREE.Raycaster();
                 var htmlInfo = document.getElementById('info');
                 htmlInfo.innerHTML = ' ';
 
-                raycaster.setFromCamera(mouse, globe.camera.camera3D);
-                // calculate objects intersecting the picking ray
-                var intersects = raycaster.intersectObjects($3dTilesLayerRequestVolume.object3d.children, true);
+                const intersects = view.pickObjectsAt(
```

Comment:
i'll be better than that goes on like this:
it will be better than that goes on like this:
```js
layer.on('mouseMove', (event) => { 
    const intersects = event.intersects;
}
```

Reply:
Feel free to implement `.on()` for layers if you think it's useful, but that's not the scope of this PR.